charts/kuberpult/environments/development/config.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
Is this the right location? run-kind.sh references `../../../infrastructure/scripts/create-testdata/testdata_template/environments`.